manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml

```diff
@@ -46,10 +46,15 @@ spec:
         args:
           - "/work"
           - "manager"
-          - "--work-driver=kube"
+          - "--work-driver={{ .WorkDriver }}"
+          {{ if eq .WorkDriver "kube" }}
```

Comment:
should be like this ?
```
{{ if .HostedMode }}
{{ if eq .WorkDriver "kube" }}
```
hostedMode only in the case` workDirver == kube`?

Reply:
Currently hosted mode only apply in kube work driver.